opentelemetry-api/tests/__init__.py

```diff
@@ -11,3 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import pkg_resources
+
+# naming the tests module as a namespace package ensures that
+# relative imports will resolve properly for other test packages,
+# as it enables searching for a composite of multiple test modules.
+#
+# only the opentelemetry-api directory needs this code, as it is
+# the first tests module found by pylint during eachdist.py lint
+pkg_resources.declare_namespace(__name__)
```

Comment:
this can be removed once this bug is fixed / clarified: https://github.com/PyCQA/pylint/issues/3609